[IMP] Livechat: URL Regex clarification (#10658)
odoo/documentation: content/applications/websites/livechat.rst

Live Chat
=========

Odoo **Live Chat** allows users to communicate with website visitors in real-time. With **Live
Chat**, leads can be qualified for their sales potential, support questions can be answered quickly,
and issues can be directed to the appropriate team for further investigation or follow up. **Live
Chat** also provides the opportunity for instant feedback from customers.

Enable Live Chat
================

The **Live Chat** application can be installed multiple ways:

> - Go to :menuselection:`Apps application --> Live Chat` and click :guilabel:`Install`.

Reviewer: ```suggestion
- Go to :menuselection:`Apps application`, search `Live Chat`, and click :guilabel:`Install`.
```

     :alt: View of the settings page and the live chat feature for Odoo Live Chat.

.. note::

>    After the **Live Chat** application is installed, a live chat *Channel* is created by default.

Reviewer: ```suggestion
   After the **Live Chat** application is installed, a live chat *Channel* is created, by default.
```

   modal, adjust any information, as needed. Then, click :guilabel:`Save`, or click
   :guilabel:`Remove` to remove that operator from the channel.

Click :guilabel:`Add` to reveal an :guilabel:`Add: Operators` pop-up window.

In the pop-up window, scroll to find the desired users, or enter their name in the search bar. Then,
tick the checkbox next to the users to be added, and click :guilabel:`Select`.

> New operators can be created and added to the list directly from this modal, as well, by clicking

Reviewer: ```suggestion
New operators can be created and added to the list directly from this pop-up window, as well, by clicking
```
opting for this phrasing because it was already used above (as opposed to modal). Just to keep things uniform throughout...

.. _livechat/livechat-button:

> Livechat button section

Reviewer 2: Adding "section" in the headings over and over across all docs is unnecessary. It's implied since it's a mid-tier heading that it's a section label; I think the way you were doing it before Jess was better

Reviewer 2: cc: @ksc-odoo

Reviewer: @StraubCreative gotcha. I'll be sure to not leave such a suggestion moving forward. Thanks for pointing this out. Really appreciate it. 🙏

Create new rules
~~~~~~~~~~~~~~~~

Fill out the fields on the :guilabel:`Create Rules` pop-up window as instructed below, then click
:guilabel:`Save & Close`.

.. tabs::

   .. tab:: Live Chat Button

      The *Livechat Button* is the icon that appears in the bottom-right corner of the website.
      Select from one of the following display options:

      - :guilabel:`Show`: displays the chat button on the page.
      - :guilabel:`Show with notification`: displays the chat button, as well as a floating text
        bubble next to the button.

>       - :guilabel:`Open automatically`: displays the button and automatically opens the chat window

Reviewer: ```suggestion
      - :guilabel:`Open automatically`: displays the button, and automatically opens the chat window
```
